### ISSUE TITLE
Interaction analyses

### ISSUE THREAD

**Issue by swvanderlaan** [corbinq/apex#3]
Are you working on an implementation to enable interactions?

**Comment by azodichr** [comment]
Interaction terms would be a very useful function for my group as well!

**Comment by davismcc** [comment]
Yes, I would also love to see this feature. Impressive software :D

**Comment by corbinq** [comment]
Thanks for the question -- GxE / GxG is not currently supported, but would
be a great feature.  Hopefully we can add it in the future.
